pynucastro/reduction/drgep_module.py

```diff
@@ -289,32 +330,58 @@ def _drgep_mpi_numpy(net, conds, targets, tols):
 
 
 def drgep(net, conds, targets, tols, returnobj='net', use_mpi=False, use_numpy=False):
-    """Implement Directed Relation Graph with Error Propagation
-    (DRGEP) reduction method described in
-    :cite:t:`pepiot-desjardins:2008` and :cite:t:`niemeyer:2011`.
-
-    :param net: The network (RateCollection) to reduce.
-    :param conds: A set of conditions to reduce over. Should either be a sequence of (composition,
-        density, temperature) sequences/tuples if running in standard mode, or a sequence of 3
-        sequences ((composition, density, temperature) ordering) if running in NumPy mode. In the
-        latter case, the sequences will be permuted to create the dataset. The compositions should
-        be pynucastro Composition objects.
-    :param targets: A collection of target nuclei (or a single target nucleus) to run the
-        graph search algorithm from. Should be supplied as pynucastro Nucleus objects or strings.
-    :param tols: Tolerance(s) or cutoff threshold(s) to use for paths from each of the target nuclei.
-        Nuclei whose interaction coefficients do not meet the specified tolerance will have their
-        interaction coefficients set to 0.0. Can be a single number (will be the same for all targets)
-        or a separate value for each target nucleus.
-    :param returnobj: The type of object to return. Options are 'net' (a reduced network, the default
-        setting), 'nuclei' (unique nuclei with nonzero interaction coefficients, ordered so the
-        interaction coefficients are descending), and 'coeff' (the interaction coefficients as a
-        NumPy array, with entries corresponding to nuclei in *net.unique_nuclei*).
-    :param use_mpi: Whether to divide up the set of conditions across MPI processes or not. Default
-        setting is *False*.
-    :param use_numpy: Whether to use NumPy to vectorize the interaction coefficient calculations or
-        not. This is more memory intensive and may actually hinder performance for some setups.
-        Conditions should be supplied as 3 lists that will be permuted to form the dataset (see
-        *conds* parameter). Default setting is *False*.
+    """Apply the Directed Relation Graph with Error Propagation
+    (DRGEP) reduction method to a network.  This implementation is
+    based on the description in :cite:t:`pepiot-desjardins:2008` and
+    :cite:t:`niemeyer:2011`.
+
+    Parameters
+    ----------
+    net : RateCollection
+        The network to reduce (can be a subclass of ``RateCollection``)
+    conds : Iterable
+        The set of thermodynamic conditions to reduce over. This can be:
+
+        * a sequence of tuples: (``Composition``, density,
+          temperature) if running in the default mode
+
+        * a sequence of 3 sequences ((``Composition``, density,
+          temperature) ordering) if running in NumPy mode. In this
+          case, the sequences will be permuted to create the
+          dataset. The compositions should
```

Comment:
Seems like text is being cutoff in the end here

Reply:
fixed -- I was going to say it should be a `Composition` object, but I tried to note that in the tuple itself.